src/components/DeveloperDocsLinks.tsx

```diff
@@ -15,10 +15,10 @@ const DeveloperDocsLinks = ({ headerId }: DeveloperDocsLinksProps) => (
     {docLinks
       .filter(({ id }) => id.includes(headerId))
       .map(({ items, id }) => (
-        <UnorderedList ms={6} spacing={3} key={id}>
+        <ul className="ms-6 space-y-3" key={id}>
```

Comment:
Lets replace this with the `UnorderedList` and `ListItem` defined in [this file](https://github.com/ethereum/ethereum-org-website/blob/dev/src/components/ui/list.tsx)

Reply:
Got it.